.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -0,0 +1,112 @@
+      description: A clear and concise description of what the bug is.
+    validations:
+      required: true
+  - type: textarea
+    id: expected-behavior
+    attributes:
+      label: Expected Behavior
+      description: A clear and concise description of what you expected to happen.
+    validations:
+      required: true
+  - type: markdown
+    attributes:
+      value: |
+        ---
+        #### This part is incredibly important:
+
+        Please run the following shell command in the OpenROAD root folder:
+        ```
+        ./etc/Env.sh
+        ```
+        If you are an OpenLane user, please run the following shell command in the OpenLane root folder:
+        ```
+        python3 ./env.py issue-survey
+        ```
+        Then copy and paste the ENTIRE output in the text field below.
+
+        *If the command **does not succeed**, you are using an out-of-date version of OpenROAD, and it is recommended that you update.*
+  - type: textarea
+    id: environment
+    attributes:
+      label: Environment
+      render: shell
+      description: Please paste the output from the previous command. It will be automatically formatted into code, so no need for backticks.
+    validations:
+      required: true
+  - type: markdown
+    attributes:
+      value: |
+        ---
+        #### To reproduce the bug: 
+        ##### If you are running from OpenLane, please run the following shell command in the OpenLane root folder:
+        ```
+        python3 ./scripts/or_issue.py\
+            --tool openroad\
+            --script ./scripts/openroad/<script-name>\
+            <run-path> # e.g designs/spm/runs/RUN_2022.03.01_19.21.10/tmp/routing/17-fill.def
+            # run path is implicitly specified by input def
+        ```
+        *For more information, follow this link https://openlane.readthedocs.io/en/latest/for_developers/using_or_issue.html*
+
+        ##### Otherwise, you have two options here:
+
+        A. Use `make <SCRIPT_NAME>_issue` to create a tar file with all the files to reproduce the bug(s).
+
+        &nbsp;&nbsp;&nbsp;&nbsp;Steps:
+
+        &nbsp;&nbsp;&nbsp;&nbsp;1. Clone/Use [OpenROAD-flow-scripts](https://github.com/The-OpenROAD-Project/OpenROAD-flow-scripts.git) and make sure it is using the corresponding version of OpenROAD in path "OpenROAD-flow-scripts/tools/OpenROAD"
+
+        &nbsp;&nbsp;&nbsp;&nbsp;2. Set the ISSUE_TAG variable to rename the generated tar file
+
+        &nbsp;&nbsp;&nbsp;&nbsp;3. Run `make {script}_issue` in the directory "OpenROAD-flow-scripts/flow" where script is wildcarded from the "OpenROAD-flow-scripts/scripts" directory e.g `make cts_issue`
+                                          
+        &nbsp;&nbsp;&nbsp;&nbsp;4. Upload the generated tar file
+
+        B. Upload relevant files
+
+        &nbsp;&nbsp;&nbsp;&nbsp;- Upload a tar file containing the relevant files (.def, .lef and flow.tcl).
+
+        &nbsp;&nbsp;&nbsp;&nbsp;- List the commands used.
```

Comment:
I think we can have different entries here. One for "Standalone OR", "With OL", "With ORFS",...